README.md

```diff
@@ -1,25 +1,33 @@
 ###
+
 ## smart_bills
+
 ###
 
-## excalidraw link for draft 
+## excalidraw link for draft
+
     https://excalidraw.com/#room=adeb0ba335d66f97559a,ho36IK_Ko_HKeGB_A7EPJg
 
-## start the dev
-    npm run start
+## Run front-end and back-end
+
+    npm run dev
```

Comment:
added one command that can run the front-end and back-end. However you will get a Mongoose connect error but the database is still working. Need to create a new issue to fix this.

![Screenshot 2022-01-10 185717](https://user-images.githubusercontent.com/61256722/148857818-3a9f2d90-fe7a-40d7-af9f-5adb326a468f.png)